### PR TITLE
Add a note that TiUP Playground doesn't support M1

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add a note that TiUP Playground doesn't support Apple M1 so far.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: quick-start-with-tidb.md [hunk 1]
```diff
@@ -18,6 +18,10 @@ title: TiDB 数据库快速上手指南
 - 适用场景：利用本地 Mac 或者单机 Linux 环境快速部署 TiDB 集群。可以体验 TiDB 集群的基本架构，以及 TiDB、TiKV、PD、监控等基础组件的运行。
 - 耗时：1 分钟
 
+> **注意：**
+> 
+> 本地 Mac 环境下，TiUP Playground 命令暂不支持 Apple M1 芯片。
```
Comment (TomShawn) [inline quick-start-with-tidb.md:23]:
```suggestion
> 本地 Mac 环境下，暂不支持在使用 Apple M1 芯片的机器上执行 `tiup playground` 命令。
```

Comment (yikeke) [reply quick-start-with-tidb.md:23]:
```suggestion
> 暂不支持在使用 Apple M1 芯片的本地 Mac 机器上执行 `tiup playground` 命令。
```
能否简单说明下用户不支持使用的原因？

Comment (lilyjazz) [reply quick-start-with-tidb.md:23]: /lgtm

Comment (lilyjazz) [reply quick-start-with-tidb.md:23]: 因为 Playground命令 要调用其他TiDB组件来运行，但是部分组件还没有发布支持 M1 的版本，因此暂时跑不起来。
等引用的组件都可以在 M1 运行就可以使用了。

Comment (yikeke) [reply quick-start-with-tidb.md:23]: 我改了下，可以看下目前的版本可不可以

**Pull request merged**